docs/user-interfaces/elements/drop-target.mdx

```diff
@@ -103,7 +103,8 @@ Event listener for the [`drop` event][].
 #### `target`
 
 DOM element, CSS selector, or plugin to place the drag and drop area into
-(`string`, `Element`, `Function`, or `UIPlugin`, default: `null`).
+(`string`, `Element`, `Function`, or `UIPlugin`, default:
+[`Dashboard`](/docs/dashboard)).
```

Comment:
Maybe not ideal with `drop-target`, because you usually want the whole `document.body` or a specific area to accept drag-drop?

Reply:
That's a copy paste error, good catch